approx_dates/tests/test_creation.py

```diff
@@ -60,3 +60,11 @@ def test_past(self):
         assert d.earliest_date == date(1, 1, 1)
         assert d.latest_date == date(1, 1, 1)
         assert text_type(d) == 'past'
+
+    def test_future_past_immutable(self):
```

Comment:
I think these should be two tests - people vary a bit in how to interpret the ["arrage - act - assert"](http://wiki.c2.com/?ArrangeActAssert) pattern for testing, but I think there should only be single action for each test. (Or a logically grouped set of steps that you might regard as an action, etc. etc.) Anyway, in this case I'd make the past and future cases different test methods.